crates/rpc/rpc-types/Cargo.toml

```diff
@@ -38,5 +38,11 @@ bytes.workspace = true
 serde_json.workspace = true
 
 [features]
-default = ["jsonrpsee-types"]
+default = [
```

Comment:
let's rename this full and make feature full default

Reply:
Done!